tasks/copy_to_archive/copy_to_archive.py

```diff
@@ -10,10 +10,10 @@
 
 # Third party libraries
 import boto3
-from boto3.s3.transfer import MB, TransferConfig
-from run_cumulus_task import run_cumulus_task
 from aws_lambda_powertools import Logger
 from aws_lambda_powertools.utilities.typing import LambdaContext
+from boto3.s3.transfer import MB, TransferConfig
+from run_cumulus_task import run_cumulus_task
```

Comment:
I thought run_cumulus_task was removed from all Lambdas per ORCA-522. If we decided to hold off on this we should create a card in the backlog and put a note here so that we do not forget to remove it in future releases.

Reply:
copy_to_glacier/copy_to_archive requires an additional adapter layer. https://bugs.earthdata.nasa.gov/browse/ORCA-520